content/21.nutraceuticals.md

```diff
@@ -178,22 +272,56 @@ However, in the case of probiotics and respiratory infections, the evidence seem
+Certainly, randomized controlled trials are required to investigate these findings further.
+
+In this review, we report the findings to date for several dietary supplements and nutraceuticals.
+While existng evidence suggests potential benefits of n-3 PUFA and probiotic supplementation for COVID-19 treatment and prophylaxis, clinical data is lacking but trials are underway.
+Both zinc and vitamin C supplementation in hospitalized patients seem to be associated with positive outcomes, however, further clinical trials are required. 
+In any case, both vitamin C and zinc intake are part of a healthy diet and both likely presents minimal risk when supplemented for, but their potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
+On the other hand, mounting evidence from observational studies indicates that there is an association between vitamin D deficiency and COVID-19 incidence, which is supported by meta-analysis [@doi:10.1080/10408398.2020.1841090].
+Indeed, scientists are working to affirm these findings and to determine whether a patients serum 25-hydroxyvitamin D levels are also associated with COVID-19 severity.
+Clinical trials are required to determine whether preemptive vitamin D supplementation may mitigate against severe COVID-19.
+In terms of the therapeutic potential of vitamin D, initial evidence from clinical trials are conflicting, but seem to indicate that vitamin D supplementation may reduce COVID-19 severity [@doi:10.1016/j.jsbmb.2020.105751].
+The various clinical trials currently underway will be imperative to provide information on the efficacious use of vitamin D supplementation for COVID-19 prevention and/or treatment.
+
+Finally, it is known that a patient's nutritional status affects health outcomes in various infectious diseases [@doi:10.3390/nu12041181], and COVID-19 is no different [@doi:10.1016/j.clnu.2020.03.022; @doi:10.1093/advances/nmaa125; @doi:10.1016/j.clnesp.2020.09.214]. 
+Some of the main risk factors for severe COVID-19, which also happen to be linked to poor nutritional status includes obesity, hypertension, cardiovascular diseases, type II diabetes mellitus, and indeed age-related malnutrition [@doi:10.1093/advances/nmaa125; @doi:10.3390/nu12051466; @doi:10.1002/jpen.1954].
+Although not the main focus of this review, it is important to consider the nutritional challenges associated with severe COVID-19 patients.
+Hospitalized COVID-19 patients tend to report an unusually high loss of appetite preceeding admission, some suffer diarrhea and gastrointestinal symptoms in hospital resulting in significantly lower food intake, and patients with poorer nutritional status were more likely to have worse outcomes and require nutrition therapy [@doi:10.1002/jpen.1953].
+Dysphagia also seems to be a significant problem in rehabilitating COVID-19 patients, potentially contributing to poor nutritional status [@doi:10.1038/s41430-020-0664-x]. 
```

Comment:
In case it's useful information, pediatric patients also sometimes have difficulty swallowing (it seems to be more associated with MIS-C)

Reply:
Good shout! I will put something in here.